README.md

```diff
@@ -250,6 +250,27 @@ $ remme public-key get-info \
 }
 ```
 
+Get a particular state by its address — ``remme state get``:
```

Comment:
Why `particular` here? Code docs do not have this word.

Reply:
@dmytrostriletskyi, done.